crates/eww/src/error_handling_ctx.rs

```diff
@@ -31,9 +31,6 @@ pub fn print_error(err: anyhow::Error) {
 }
 
 pub fn format_error(err: &anyhow::Error) -> String {
-    for err in err.chain() {
-        format!("chain: {}", err);
-    }
```

Comment:
i'm guessing this was some leftover debug print
removing it seemed to make the most sense here